src/utils.rs

```diff
@@ -183,6 +183,9 @@ pub(crate) fn open_database(wallet_opts: &WalletOpts) -> Result<AnyDatabase, Err
             .into_string()
             .expect("path string"),
     });
+    #[cfg(not(any(feature = "key-value-bd", feature = "sqlite-db")))]
```

Comment:
A little typo to fix here:
```suggestion
    #[cfg(not(any(feature = "key-value-db", feature = "sqlite-db")))]
```

Reply:
PS. I updated the github checks to match the new workflow matrix.

Reply:
Ohhh this would have been horrible to debug, thanks!